rust/common/error/macros.rs

```diff
@@ -168,7 +168,6 @@ macro_rules! error_messages {
 
 #[cfg(test)]
 mod tests {
```

Comment:
Drive-by: this needs `#[allow(dead_code)]` so that rust doesn't generate warnings for the tests-only generated code.